api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -116,4 +116,8 @@
 		<addForeignKeyConstraint constraintName="medication_dispense_voided_by_fk" baseTableName="medication_dispense" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id"/>
 	</changeSet>
 	
+	<changeSet id="TRUNK-6081-2022-07-10" author="wikumChamith">
+		<addNotNullConstraint tableName="encounter_diagnosis" columnName="dx_rank" columnDataType="int"/>
```

Comment:
You are missing a comment tag, please include one to describe what the changeset does

Reply:
I added a comment to describe the changeset in the new commit.

Reply:
@wluyima could you review the changes?

Reply:
You are a missing a comment

Reply:
> You are a missing a comment

 I added a comment with the new commit.

Reply:
I don't see it

Reply:
Thanks! Looks good!